angular.json

```diff
@@ -7,10 +7,114 @@
 			"root": "",
```

Comment:
ggf. könnten wir das project mal umbenennen. terra-components finde ich für die Spielwiese irgendwie ungünstig.
ggf. in dev-app oder so

Reply:
Bin dran

Reply:
https://github.com/plentymarkets/terra-components/pull/1024